Mathlib/CategoryTheory/FiberedCategory/Fiber.lean

```diff
@@ -0,0 +1,108 @@
+/-
+Copyright (c) 2024 Calle Sönne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Calle Sönne, Paul Lezeau
+-/
+
+import Mathlib.CategoryTheory.FiberedCategory.HomLift
+import Mathlib.CategoryTheory.Functor.Const
+
+/-!
+
+# Fibers of a functors
+
+In this file we define, for a functor `p : 𝒳 ⥤ 𝒴`, the fiber categories `Fiber p S` for every
+`S : 𝒮` as follows
+- An object in `Fiber p S` is a pair `(a, ha)` where `a : 𝒳` and `ha : p.obj a = S`.
+- A morphism in `Fiber p S` is a morphism `φ : a ⟶ b` in 𝒳 such that `p.map φ = 𝟙 S`.
+
+For any category `C` equipped with a functor `F : C ⥤ 𝒳` such that `F ⋙ p` is constant at `S`,
+we define a functor `InducedFunctor : C ⥤ Fiber p S` that `F` factors through.
+-/
+
+universe v₁ u₁ v₂ u₂ v₃ u₃
+
+open CategoryTheory Functor Category IsHomLift
+
+variable {𝒮 : Type u₁} {𝒳 : Type u₂} [Category.{v₁} 𝒮] [Category.{v₂} 𝒳]
+
+/-- `Fiber p S` is the type of elements of `𝒳` mapping to `S` via `p`.  -/
+def Fiber (p : 𝒳 ⥤ 𝒮) (S : 𝒮) := { a : 𝒳 // p.obj a = S }
```

Comment:
Is this in the root namespace? I think it should be in `CategoryTheory.Functor`.

Reply:
good point, thanks a lot!